django_ilmoitin/templates/admin/preview_template.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'admin/change_form.html' %}
```

Comment:
please, make sure all the indents are consistent. now you have 2, 4 and 8 spaces indents. maybe make all of them 4 spaces?